### ISSUE TITLE
Reduce height of hero

### ISSUE THREAD

**Issue by edmullen** [18F/fec-style#330]
# Summary

_This is a suggestion. I'm dropping it here to not lose track of it._

On the legal side, I've observed users missing the content below the hero because it outside of the viewport on desktop. I think the visual impact of the hero could be retained while reducing the height to provide greater indication that content is available below. 

We're using the primary hero styling because I wanted consistency with the other second levels. I think reducing the height some could prove beneficial across the site, so I'm pitching that idea here. 

cc @jenniferthibault @manger @adborden

**Comment by jenniferthibault** [comment]
Seeing similar patterns on the data side, and our [latest mockups](https://gsa.invisionapp.com/share/836FTMS6F#/screens/152007673_1-1_Landing) for the data landing page also hint toward a smaller hero space. Looking at work @emileighoutlaw is doing on the copy for the Reg/Reporting side also supports a faster experience over the hero area. 

I'm assigning this to myself, and backlogging it until next sprint.

**Comment by edmullen** [comment]
I wanted to follow up on this after a couple more rounds of testing that indicated a shorter hero would help people see the down-screen components better on desktop. Specifically, on the legal resources landing page, moving the nav and content area higher will help people understand that the page is not only about search, though that should remain prominent.

![image](https://cloud.githubusercontent.com/assets/2654503/15153402/2281ff48-16a7-11e6-9de5-00180da97341.png)

**Comment by jenniferthibault** [comment]
Alrighty, I have a redesign for the hero template that I'm excited to show you all. First, per usual, I will make you read through some thoughts on the rationale behind the changes 😁 🚨 long post alert 🚨 

- My goal for this template is to keep a recognizable, distinct feel to the top of each main section's landing page. This will help give users a sense of their position within the site, and I hope to create a sense of "intro to something"-ness, when a user recognizes this pattern. 

- For heroes that direct the user toward an action (like the search on Data and Legal), changing the layout from vertically centered and stacked to side-by-side columns and flush left allows us to both introduce the section overall and introduce the targeted interaction. Side-by-side also allows us to save vertical space.  

- I'd like to introduce this slice of a textural images to the hero template (photo is still a placeholder for now, but would be unique for each!) purely with an aesthetic motivation. One of our early visual identity design goals was to encourage information to be visually presented, through photos and graphics. So far, we're heavy on the graphics. While these images won't add _narrative_ to the design of these pages, I think they will help us nudge the design maturity forward a bit, and will ease the transition between future pages, which could be more apt for photos. Having a small slice of image here will help tie these in better. (And I think it really livens things up without too much vertical space sacrificed. It should be about the height of the breadcrumb space).

- Move the "What does it mean to register and report" text up to the hero intro area of the Reg/Reporting page: we are able to jump right into the content faster. We lose a place for the "Already registered?" search box, but would love to think through alternative placement together later on. It also addresses [this issue](https://github.com/18F/fec-style/issues/339) on balancing the space on that page. Thoughts @emileighoutlaw ? 
  -  ![screen shot 2016-05-17 at 8 12 17 pm](https://cloud.githubusercontent.com/assets/11636908/15343553/b212bf16-1c6b-11e6-83e0-3a6f740f8774.png)

- Legal resources: Dropping the "We’re designing a new way to get legal resources." alert from the middle of the page allows us to keep a consistent template. I'm wondering if you'd be up for this @edmullen , since there are other contextual ways to give feedback?
  - ![screen shot 2016-05-17 at 8 11 28 pm](https://cloud.githubusercontent.com/assets/11636908/15343543/96c52924-1c6b-11e6-82f7-9ddcb966505f.png)

- Note: all copy that I've inserted is for placement only! Would love @emileighoutlaw 's thought on the hero intros for each section. 

### Desktop template

![hero-desktop-data](https://cloud.githubusercontent.com/assets/11636908/15337395/82388e8c-1c49-11e6-940f-2c402a5f4a09.png)

![hero-desktop-regreport](https://cloud.githubusercontent.com/assets/11636908/15337396/823c5490-1c49-11e6-9988-d04dbf5725d4.png)

![hero-desktop-legal](https://cloud.githubusercontent.com/assets/11636908/15337394/823412ee-1c49-11e6-8be7-a2a43e23a5fd.png)

### Tablet template

![hero-tablet-legal](https://cloud.githubusercontent.com/assets/11636908/15337406/95e92c52-1c49-11e6-9cd0-42e9fe756cb3.png)


### Mobile template

![hero-mobile-data](https://cloud.githubusercontent.com/assets/11636908/15337417/a13e7f94-1c49-11e6-95f9-0249f9803369.png)

And because it's hard to see the real size here, [here is your link to an InVision display](https://invis.io/K37CE66WH#/159638080_Hero-Desktop-Data)! Feel free to comment directly on that, and/or discuss here.

cc @noahmanger @emileighoutlaw @edmullen @adborden

**Comment by emileighoutlaw** [comment]
Yay, Jen!

#### Already registered box
I'm still not convinced this has been a useful addition to this page (especially now that people will be able to search candidates and committees from the mega menu).

Can we validate or invalidate my hunch with analytics?

#### Content thoughts
I like the consistency of language of "Find details on..." that we have in Legal and Campaign finance. 

##### Campaign finance
I'm eyeing the introductory sentence now that you've moved it to the same level as the search box— it feels both broad _and_ hard to parse somehow.

Our search bar already has "candidates and committees" in the placeholder. Could we winnow this sentence to something like:
> Find details about fundraising and spending in federal elections.
(I'm feeling a little subversive here, but I'm wondering what would happen if this text even went away completely? With the header and the placeholder text, I feel like we've already pretty much conveyed what's in the intro sentence?)

Speaking of placeholder text, the word "profile" is throwing me— I like it in our mega-menu, but it doesn't feel like the right word to have in this box. Could it be:

> Search candidates and committees



##### Law
Does this encompass all of what our legal section should do?
> Find details on the laws and regulations governing campaign finance activities

I think it's meatier than this, right? It's law, regulation, court cases, Commission opinions, Commission rules, enforcement actions

**Comment by edmullen** [comment]
Thanks for all this @jenniferthibault. I appreciate the narrative.

**PHOTO**

I agree the photo treatment enhances the overall aesthetic and makes it feel more rich — rich like chocolate, not rich like affluent.

**ALREADY REGISTERED BOX**

In reading about this in the other issue, I was thinking this could be accomplished inline within the body as a simple link to the committee search page. Almost like an "Already have an account?" link on a signup page. So instead of a fixed module, where appropriate just include "Already registered? [Find your committee](https://beta.fec.gov/data/committees/)" link. So that would be in support of removing, though my assessment is just intuition, not based on users.

**SEARCH**

Retaining the search within the hero space concerns me for a couple reasons. 

**1. Mixing concerns**

Now that the hero is smaller, it feels a bit like we're mixing concerns, positioning the user within the site _and_ promoting a first action. Given the reduced space, this feels more tense now.

**One possible route:** I wonder if search could be shifted into the white space of the page right below the hero, similar to this, but with reduced negative space:

![image](https://cloud.githubusercontent.com/assets/2654503/15393068/3483e5c8-1d99-11e6-9de6-3c614927a18f.png)

I think that could be made to work well with the new results format:

![image](https://cloud.githubusercontent.com/assets/2654503/15393099/69694c6a-1d99-11e6-91de-2311717d91dc.png)

**2. Unified search**

I think eventually there's going to be a need to grapple with _search across all the things._ Not because users will be looking for content from multiple sections, but because the won't know which one to choose prior to a localized search. (I'm not suggesting we should abandon localized search.) Once all the prose content @emileighoutlaw is working on launches, I think there will be a real need for users to be able to skip navigation and find what they need through search. The [new results patterns](https://github.com/18F/openFEC-web-app/issues/1202) are not in conflict with this. Fixing the search within the hero feels less flexible and more brittle now.

So I fully understand the role of a local search at the _content type_ level and don't see that ever going away...

![image](https://cloud.githubusercontent.com/assets/2654503/15393643/7754a0b0-1d9c-11e6-9360-83d471dd4ae9.png)

...but I'm less clear on search at the major section level, where on the data side, it's essentially providing the ability to search _some_ of the content within the section. This is sort of an incomplete thought. I'll think about it a bit more and follow up.

**Comment by jenniferthibault** [comment]
#### On search
Separating the search from the hero space makes a lot of sense to me: in our current version, where the hero space takes up so much vertical room, you need to embed an action or else it's not clear what you can do. In this template, with a shorter hero, more of the page's functionality will be visible, so we can separate the intro from the first interaction. To keep clear of universal search vs search within each section, I think adding the section's icon within the search bar is helpful. I made a quick icon for the legal side to demonstrate the concept, but we can keep working on that separately.

#### On making the entire landing pages identical
I do not think that all landing pages need to be totally identical in format because they're all serving different goals right now, and the designs reflect that. It's nice to build parity when goals are close, like Reg/Reporting & Legal, but no need to force it when it doesn't fit. (So no need for a side menu on the data page). For now, the goal of the design is to let users realize they are on a section landing page using just the hero space alone. If we do it well, and don't send mixed messages on other pages of the site, I believe it will work.

#### On content
If it takes some thinking & exploring, @emileighoutlaw I'd like to open a new issue for to work out of specifically for that. I'm finding it hard to track design and content feedback for this all in one place, even though they are certainly related. https://github.com/18F/fec-style/issues/356

Here's [an InVision display](https://invis.io/K37CE66WH#/160989613_Hero-Desktop-Data_Copy) with the new mockups, and also copied below. The pink is to signify that that part of the page is not part of the hero template. If you're all 👍  with this concept, I'd like to move this forward to implementation, and find photos for each section.

![hero-desktop-data copy](https://cloud.githubusercontent.com/assets/11636908/15472688/74b143a0-20ca-11e6-996a-9cc80de17d3d.png)


![hero-desktop-regreport copy](https://cloud.githubusercontent.com/assets/11636908/15472695/7903228e-20ca-11e6-99df-d9e475f6fd7e.png)


![hero-desktop-legal copy](https://cloud.githubusercontent.com/assets/11636908/15472696/7b1090ac-20ca-11e6-8422-a534b9f5e86f.png)

**Comment by noahmanger** [comment]
That all makes a lot of sense to me. Thanks!

**Comment by edmullen** [comment]
I like all this too!

**Comment by jenniferthibault** [comment]
Thanks for the feedback, all! Moving it to implementation in https://github.com/18F/fec-style/issues/370

**Comment by noahmanger** [comment]
Closing this since it moved to implementation.
